src/components/FormElements/AssetPicker.tsx

```diff
@@ -196,6 +199,7 @@ export const AssetPicker = ({
           error: assetError?.issuer || "",
         }}
         disabled={disabled}
+        hasCopyButton={isReadOnly}
```

Comment:
Since `AssetPicker` is a generic component, I think we should have a dedicated prop `hasCopyButton`, and not tie it to the `isReadyOnly` prop. There might be cases where we want to have a copy button for a picker that is not ready only.